atomate/vasp/firetasks/run_calc.py

```diff
@@ -241,6 +242,60 @@ def run_task(self, fw_spec):
         runner.run(path_dir=os.getcwd())
 
 
+@explicit_serialize
+class RunAmset(FiretaskBase):
+    """
+    Run Amset directly. Requires vasprun.xml to be in current dir.
```

Comment:
There should be an option to give path to the vasprun.xml file

Reply:
(unless this is a BoltzTrap limitation, in which case maybe write in the docs that this is the reason)